techniques/failures/F108.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 <head>
-      <title>Failure of Success Criterion 2.5.7 due to not providing a single pointer method for the user to operate a function that does not require a dragging movement</title>
+      <title>Failure of Success Criterion 2.5.7 Dragging Movements due to not providing a single pointer method for the user to operate a function</title>
```

Comment:
I do think the "does not require dragging" is important ... as otherwise this loses its meaning - if the single pointer method currently there DOES rely on dragging, then that's the failure. it's not whether or not it's single pointer that's the issue.

maybe better ...
```suggestion
      <title>Failure of Success Criterion 2.5.7 Dragging Movements due to not providing a single pointer method that does not require a dragging movement</title>
```